crates/turbopack-core/src/chunk/mod.rs

```diff
@@ -407,7 +409,7 @@ async fn chunk_content_internal<I: FromChunkableAsset>(
                 for asset in assets
                     .await?
                     .iter()
-                    .filter(|asset| processed_assets.insert(**asset))
+                    .filter(|asset| processed_assets.insert((chunking_type, **asset)))
```

Comment:
This causes `Placed` and `PlacedOrParallel` to generate two chunk items.

Can we add a `processed_assets.insert((Placed, **asset))` into the placed code path in `PlacedOrParallel` and a `processed_assets.insert((Parallel, **asset))` into the parallel code path.